src/libs/actions/Workflow.ts

```diff
@@ -280,13 +287,14 @@ function setApprovalWorkflowApprover(approver: Approver, approverIndex: number,
 
     // Check if the approver forwards to other approvers and add them to the list
     if (policy.employeeList[approver.email]?.forwardsTo) {
-        const personalDetailsByEmail = lodashMapKeys(personalDetails, (value, key) => value?.login ?? key);
```

Comment:
How come this is removed? Won't `personalDetailsByEmail` be undefined on the next line?

Reply:
@tgolen personalDetailsByEmail is defined above in line 40

Reply:
Oh, got it. I did not look high enough.